alpine.dockerfile

```diff
@@ -2,12 +2,4 @@
 # Eventually should be automated within the CI
 FROM node:10.15.1-alpine
 
-RUN apk update
-RUN apk add bash
-RUN apk add gcc
-RUN apk add g++
-RUN apk add ccache
-RUN apk add git
-RUN apk add make
-RUN apk add python
-RUN apk add linux-headers
+RUN apk update && apk add bash gcc g++ ccache git make python linux-headers
```

Comment:
You don't do ``apk update``.
You should probably add ``--no-cache --no-progress`` to ``apk add``.